plugin/build.gradle

```diff
@@ -31,6 +31,7 @@ thirdPartyAudit.enabled = false
 configurations.all {
     // conflict with spring-jcl
     exclude group: "commons-logging", module: "commons-logging"
+    exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
```

Comment:
I assume this is supposed to be same `resolutionStrategy.force` as above? Otherwise, although all IT passed, our plugin will have problem at runtime. Please confirm and do sanity test by running all ITs with remote ES cluster with our plugin built from your branch.

Reply:
Reference: `./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Dtests.class="*SQLCorrectnessIT" -Dhttps=true -Duser=admin -Dpassword=admin` in https://github.com/opendistro-for-elasticsearch/sql/pull/760